operator/pkg/securedcluster/values/translation/base-values.yaml

```diff
@@ -10,3 +10,15 @@ meta:
   useLookup: false
 allowNonstandardReleaseName: true
 allowNonstandardNamespace: true
+scannerV4:
```

Comment:
how come we didn't need this for Scanner v2?

Reply:
But isn't it here
https://github.com/stackrox/stackrox/blob/14ec4c7db56d9e1248ad0e4bfd10400b6934793f/operator/pkg/central/values/translation/base-values.yaml#L34?

Reply:
I'd suggest to track these questions with a ticket. WDYT?